maps/predship/huntership.dmm

```diff
@@ -1118,6 +1142,7 @@
 /area/yautja)
 "cC" = (
 /obj/structure/machinery/autolathe/yautja,
+/obj/structure/closet/wardrobe/chaplain_black,
```

Comment:
what is this and why is it below the autolathe?

Reply:
fixed, removed